exasol/shared_memory_vault.py

```diff
@@ -0,0 +1,226 @@
+from typing import Optional
+from multiprocessing.shared_memory import SharedMemory
+from math import ceil
+from datetime import datetime
+import struct
+
+
+DEFAULT_CRC_DIVISOR = '1001011001100101'
+DEFAULT_MAX_SIZE = 100
+DEFAULT_STORAGE_NAME = 'notebook_connector_vault'
+
+
+def _xor(sequence: str, crc_divisor: str) -> str:
+    return ''.join('0' if a == b else '1' for a, b in zip(sequence, crc_divisor))
+
+
+def compute_crc(sequence: str, crc_divisor: str) -> str:
```

Comment:
Maybe I will leave it as it is for now. The chances are it will be gone. For example, if we start using the file lock we definitely can scrap the CRC.

Reply:
:+1: